ibc-testkit/src/fixtures/clients/tendermint.rs

```diff
@@ -22,7 +22,10 @@ use typed_builder::TypedBuilder;
 pub fn dummy_tm_client_state_from_raw(
     frozen_height: RawHeight,
 ) -> Result<TmClientState, TendermintClientError> {
```

Comment:
Looks more appropriate we change the signature here to `Result<TmClientState, DecodingError>`